src/FluentCheck.ts

```diff
@@ -56,7 +58,9 @@ export class FluentCheck<Rec extends ParentRec, ParentRec extends {}> {
 
   protected run(
     testCase: WrapFluentPick<Rec> | Rec,
-    callback: (arg: WrapFluentPick<Rec> | Rec) => FluentResult): FluentResult {
+    callback: (arg: WrapFluentPick<Rec> | Rec) => FluentResult,
+    testCases: ValueResult<any>[],
+    _partial: FluentResult | undefined = undefined): FluentResult {
```

Comment:
I'm 99% certain this `_partial` was removed. Why is it being reintroduced?

Reply:
yes, my bad, I didn't notice it while merging master (should've rebased but when I noticed it was too late).